plugins/sap-concur-approve-expense-report/README.md

```diff
@@ -17,7 +17,689 @@ systems:
+
+
+# **Design Questions to Consider**
+
+1. What fields would you like to show inside the approval request notification?
+2. Which actions would you like the approver to be able to take on the notification?
+
+# **SAP Concur Configuration**
+
+Please review the [Connector Guide](https://developer.moveworks.com/creator-studio/resources/connector?id=sap-concur) to set up a new connector in Creator Studio that connects to your SAP Concur instance for using the Expense [Reports v4](https://developer.concur.com/api-reference/expense/expense-report/v4.reports.html) and [Workflows v4](https://developer.concur.com/api-reference/expense/expense-report/v4.workflows.html) APIs.
+
+
+# **For Creator Studio Developers**
+
+## **Step 1: Check if you have built a Creator Studio Connector for your middleware / APIM tool**
+
+This step was outlined in the [Prerequisites section](#prerequisites) above, and should be completed before you begin building your plugin. If you do not have a connector for your middleware tool, you can learn more about how to build one in our [Connector Configuration Guide](https://developer.moveworks.com/creator-studio/integrations/outbound/connector-configuration/).
+
+## Step 2: Test required APIs with Postman
+
+Copy the cURL commands below into your API client like Postman for testing. Substitute any `{{placeholder text}}` with relevant SAP Concur API credentials or inputs.
+
+### API 1: Authorization
+
+```bash
+curl --location 'https://us2.api.concursolutions.com/oauth2/v0/token' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'client_id={client_id}' \
+--data-urlencode 'client_secret={client_secret}' \
+--data-urlencode 'refresh_token={refresh_token}' \
+--data-urlencode 'grant_type=refresh_token'
+```
+
+### **API 2: Get All Users**
```

Comment:
It doesn't look like this gets used in this guide & we can maybe defer this to Moveworks Setup